sound/soc/bcm/audioinjector-pi-hat.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * ASoC Driver for AudioInjector Pi Hat
+ *
+ *  Created on: 13-May-2016
+ *      Author: flatmax@flatmax.org
```

Comment:
It would be polite to mention another author here, since nobody in their right minds writes modules like this without reading another. The "i2s-controller" support looks familiar, for a start.